src/platform/intel/cavs/lib/clk.c

```diff
@@ -32,7 +33,7 @@ static inline void select_cpu_clock(int freq_idx, bool release_unused)
 
 #if CONFIG_TIGERLAKE
 	if (freq_idx == CPU_HPRO_FREQ_IDX)
-		cpu_enable_core(PLATFORM_CORE_COUNT - 1);
+		pm_runtime_get(PM_RUNTIME_DSP, PWRD_BY_HPRO | (PLATFORM_CORE_COUNT - 1));
```

Comment:
A comment here would be good explaining what this does i.e. why we only do this on TGL.

Reply:
added just a small note about HW recommended flow